java/net/anotheria/anosite/content/servlet/ContentPageServlet.java

```diff
@@ -2045,9 +2047,12 @@ private Pagex getPageByName(String pageName) throws ServletException {
     private Pagex getPageByLocalizedName(String pageName) throws ServletException {
         try {
             String localizedNameProperty = "localizedName_" + ContextManager.getCallContext().getCurrentLanguage();
-            return webDataService.getPagexsByProperty(localizedNameProperty, pageName).get(0);
+            List<Pagex> pages = webDataService.getPagexsByProperty(localizedNameProperty, pageName);
+            return pages.isEmpty() ? null : pages.get(0);
         } catch (Exception e) {
-            //ignore ->
+            if (LOGGER.isTraceEnabled()) {
+                LOGGER.trace("can't find page with localized name: '" + pageName + "' return null.", e);
```

Comment:
in scope of msg : 
 LOGGER.trace("getPageByLocalizedName("+pageName+")", e) - will match better, cause You can't know what did You got in common Exception scope :P